packages/react-provider/package.json

```diff
@@ -28,6 +28,7 @@
     "@fluentui/babel-make-styles": "9.0.0-beta.2",
     "@fluentui/eslint-plugin": "*",
     "@fluentui/jest-serializer-make-styles": "9.0.0-beta.2",
+    "@fluentui/react-button": "9.0.0-beta.2",
```

Comment:
I think someone said that dev dependencies was the right place for story-only dependencies.  @ling1726?

Reply:
nope, dependencies in stories are not _**real**_ dependencies since we don't actually ship them with our packages. They should not be in the package.json